frontend/src/container/QueryBuilder/components/ToolbarActions/ToolbarActions.styles.scss

```diff
@@ -68,4 +112,17 @@
 			}
 		}
 	}
+	.loading-container {
+		.loading-btn {
+			background: var(--bg-vanilla-300);
+		}
+
+		.cancel-run {
+			color: var(--bg-vanilla-100);
+		}
+
+		.cancel-run:hover {
+			background-color: #ff7875;
```

Comment:
Do we not have this color in the design tokens?

Reply:
no we do not have this token yet

Reply:
@GeekBoySupreme are we supposed to use colors outside design tokens or are we adding this to the tokens ?